test/image_volume.bats

```diff
@@ -45,9 +49,12 @@ function teardown() {
 		skip "userNS enabled"
 	fi
 	CONTAINER_IMAGE_VOLUMES="mkdir" start_crio
+
 	pod_id=$(crictl runp "$TESTDATA"/sandbox_config.json)
-	image_volume_config=$(cat "$TESTDATA"/container_config.json | python -c 'import json,sys;obj=json.load(sys.stdin);obj["image"]["image"] = "quay.io/crio/image-volume-test"; obj["command"] = ["/bin/sleep", "600"]; obj["linux"]["security_context"]["run_as_user"]["value"] = 1000; json.dump(obj, sys.stdout)')
-	echo "$image_volume_config" > "$TESTDIR"/container_image_volume.json
+
+	jq '.image.image = "quay.io/crio/image-volume-test"| .command = ["bin/sleep", "600"] | .linux.security_context.run_as_user.value = 1000' \
```

Comment:
for the sake of readability, can you split it into multiple lines, something like this:
```suggestion
	jq '.image.image = "quay.io/crio/image-volume-test"
			| .command = ["bin/sleep", "600"]
			| .linux.security_context.run_as_user.value = 1000' \

Reply:
fixed